packages/pigeon/lib/generator_tools.dart

```diff
@@ -82,3 +82,37 @@ class Indent {
 String makeChannelName(Api api, Method func) {
   return 'dev.flutter.dartle.${api.name}.${func.name}';
 }
+
+/// Represents the mapping of a Dart datatype to a Host datatype.
+class HostDatatype {
+  /// Parametric constructor for HostDatatype.
+  HostDatatype({this.datatype, this.isBuiltin});
+
+  /// The [String] that can be printed into host code to represent the type.
+  final String datatype;
+
+  /// `true` if the host datatype is something builtin.
+  final bool isBuiltin;
+}
+
+/// Calculates the [HostDatatype] for the provided [Field].  It will check the
+/// field against the [classes] to check if it is a builtin type.
```

Comment:
if you're referring to arguments, the dartdoc should be `classes`

Reply:
done